Infrustructure/IoC/DependencyContainer.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Application.Interfaces;
+using Application.Services;
+using Infrustructure.Integration;
+using Microsoft.Extensions.DependencyInjection;
+using RestSharp;
+
+namespace Infrustructure.IoC
+{
+    public class DependencyContainer
+    {
+
+        public static void RegisterServices(IServiceCollection services)
+        {
+            services.AddScoped<IShipService, ShipService>();
```

Comment:
Why scoped life time ? why not singleton?

Reply:
Actually I wanted to use transient since I want them to have short life time so at least I can prevent memory-leak.